### PR TITLE
Add code-server icon behind `USE_CODE_SERVER_ICON`

### LINKED ISSUE #60: Allow to switch the icon to code-server icon
Currently the VS Code icon is used even when code-server is run. This is problematic because the Visual Studio Code icon is a Microsoft trademark and should not be used to represent products other than VS Code as per https://code.visualstudio.com/brand

Proposed solution:
- ship the code-server icon (https://github.com/coder/code-server/blob/main/src/browser/media/favicon.svg) as well
- allow to switch the icon based on an environment variable config

Any thoughts/counter ideas before I open a PR?

### PR DESCRIPTION
**Pull request by krassowski**
Closes #60

### CODE REVIEW

**Comment by xhochy** [comment]
Shouldn't we always use this icon?

File: jupyter_vscode_proxy/__init__.py [hunk 1]
```diff
@@ -61,14 +61,16 @@ def _get_cmd(port: int) -> List[str]:
 
 def setup_vscode() -> Dict[str, Any]:
     executable = os.environ.get("CODE_EXECUTABLE", "code-server")
+    use_code_server_icon = os.environ.get("USE_CODE_SERVER_ICON", "false").lower()
+    icon = "code-server.svg" if use_code_server_icon == "true" else "vscode.svg"
```
Comment (krassowski) [inline jupyter_vscode_proxy/__init__.py]: > Shouldn't we always use this icon?

When the executable is using `code-server` yes, but when it is `openvscode-server`... well, it is not the code-server. I wanted avoid breaking the experience for existing users, but I was also really tempted to rewrite it to:

```suggestion
    icon = "code-server.svg" if executable == "code-server" else "vscode.svg"
```

What do you think @xhochy? Feel free to commit my suggestion directly :)

**Comment by benz0li** [comment]
@krassowski Your SVG does not work for me.

The `jp-LauncherCard-icon` is embedded as `img` and seems to prevent `jp-icon0` from working.

**Comment by krassowski** [comment]
This may be something we need to change on JupyterLab side.

**Pull request merged**